tk_toolchain/cmd_line_tools/tk_build_qt_resources/__init__.py

```diff
@@ -148,6 +215,11 @@ def main():
         default="tank.platform.qt",
         help="The import text to replace",
     )
+    parser.add_argument(
+        "-y",
+        "--yamlfile",
+        help="The path to the YAML file with commands",
```

Comment:
Can we have a default name/value here so in most repos, we won't have to specify it

Reply:
Also you can specify FileType? https://docs.python.org/3/library/argparse.html#argparse.FileType

Reply:
I don't understand why we want to complicate this. I must miss something.

Reply:
Hum, I also don't understand why we want to use an absolute path here... It makes the script complicated to use. Ideally, we'd call it without any argument.

I made some tests in #66. Please have a look. I also tested the changes in:

- shotgunsoftware/tk-core#977 - works with no changes
- shotgunsoftware/tk-multi-shotgunpanel#94 - I created a brand new PR for it
- shotgunsoftware/tk-flame-review#35 - I created a brand new PR for it

I believe having a few more PRs like that will give us a good overview of what we want here.

Reply:
Now It's exists several ways to run the command after a pip installation "pip install tk-toolchain" and set PYTHONPATH with the python environment path where has to exists pyside2-uic and pyside2-rcc compilers:
```
    >  tk-build-qt-resources

    >  tk-build-qt-resources -y name_of_yml_file_with_commands.yml

    >  tk-build-qt-resources -y name_of_yml_file_with_commands.yml -p /path/to/python/env

    >  tk-build-qt-resources -y name_of_yml_file_with_commands.yml -u /path/to/pyside2-uic -r /path/to/pyside2-rcc
```

So for tk-core or any other project with resources, It will be possible to run only `tk-build-qt-resources`